src/components/Pill/Pill.tsx

```diff
@@ -1,15 +1,14 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import { ClassValue } from 'classnames/types';
 import Addon from './Addon';
 
 import styles from './Pill.module.css';
 
 export interface PillProps
   extends Omit<React.HTMLAttributes<HTMLDivElement>, 'className' | 'onClick'> {
   checked?: boolean;
-  className?: ClassValue;
+  className?: string;
```

Comment:
This line (and the Omit counterpart) can be omitted since className now matches the type already on `div`